app/jsx/components/Header2Comp.jsx

```diff
@@ -16,8 +15,10 @@ import MEDIA_PATH from '../../js/MediaPath.js'
 class HeaderComp2 extends React.Component {
   static propTypes = {
     type: PropTypes.string,       // not required, at least on global search page
+          // i.e.: "campus|journal|oru|series|monograph_series|seminar_series|special"
     unitID: PropTypes.string,     // ditto
     searchComp: PropTypes.string, // ditto
+    hideAdminBar: PropTypes.bool, // AdminBar not needed on Item page
```

Comment:
Oh, I hadn't realized it was showing. Good idea!

Reply:
Well, up until I started this task, It *wasn't* showing, because PageBase's componentDidMount (which fetches permission from sessionData) was never being triggered. But that's fixed now (calling super() from inherited class for now).